client/state/ui/guided-tours/contexts.js

```diff
@@ -26,6 +27,9 @@ export const isNewUser = state => {
 	return ( Date.now() - creation ) <= WEEK_IN_MILLISECONDS;
 };
 
+export const userHasInteractedWithComponent = componentName => state =>
+	getLastAction( state ).component === componentName;
+
```

Comment:
@mcsf Agree. Had a similar dilemma with the naming of our capability checking selector, and opted to diverge from core function naming `current_user_can` for the reasons you mentioned, naming ours `canCurrentUser`.

https://github.com/Automattic/wp-calypso/blob/8247391/client/state/current-user/selectors.js#L89-L108
https://codex.wordpress.org/Function_Reference/current_user_can